src/main/java/ru/ayaz/UserMessageDistributor.java

```diff
@@ -0,0 +1,62 @@
+package ru.ayaz;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+
+import static ru.ayaz.MessageParser.isCommand;
+
+public class UserMessageDistributor implements Runnable {
+
+    private Map<String, User> userMap;
+    private Map<String, UserSocketHandler> userSocketHandlerMap;
+    private BlockingQueue<UserMessage> messageQueue;
+    private MessageBroadcaster broadcaster;
+    private CommandExecutor executor;
+
+    UserMessageDistributor() {
+        this.userMap = new HashMap<>();
+        this.userSocketHandlerMap = new HashMap<>();
+        this.messageQueue = new ArrayBlockingQueue(500, true);
+        this.broadcaster = new MessageBroadcaster(userMap, userSocketHandlerMap);
```

Comment:
&#x1F34E; **Issue**
The objects your class depends on should be injected into it from the calling code (its Dependency Injection pattern and you would have used it if you started with TDD approach from the ground up instead of starting with something existing and refactoring it ;) ).
If you create them inside then you are testing not only your object, but your dependencies as well. Dependencies should be mocked (and tested in a separate test) in order for you to get small testable unit.